packages/@yodaos/mm/test.js

```diff
@@ -0,0 +1,46 @@
+var createHarness = require('tape').createHarness
+var test = createHarness({ autoclose: /** automatically emit test results on finish */true })
+
+test.createStream().on('data', row => {
+  row.toString().trim().split('\n').forEach(it => require('logger').syslog(`${it}\n`))
```

Comment:
In `syslog()`, it does open, write and close ops, just wonder if this could make too many file descriptors? How about just exposing syslog by following the original man, and only do open 1 connection, and close after the test is done?

Reply:
This is the manner of how the `rklog` works. Here I was simply followed that.

Reply:
Ok, we'll fix that latter.